lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
@jkeiser I have kind given on the one-shot for now and reverted back to the post and put call. Unless someone else can confirm one-shot API works in cheffish against chef12 server maybe we should just close this PR and I will resubmit only these lines:

https://github.com/poliva83/cheffish/blob/master/lib/chef/provider/chef_organization.rb#L55-L57